aptos-move/framework/aptos-framework/sources/vesting_without_staking.move

```diff
@@ -439,19 +441,24 @@ module aptos_framework::vesting_without_staking {
                 amount: shareholder_amount,
             },
         );
+
+        // remove `shareholder_address`` from `vesting_contract.shareholders`
         let shareholders = &mut vesting_contract.shareholders;
-        simple_map::remove(shareholders, &shareholder_address);
-        let shareholder_baneficiary = *simple_map::borrow(&vesting_contract.beneficiaries, &shareholder_address);
-        simple_map::remove(&mut vesting_contract.beneficiaries, &shareholder_baneficiary);
-        assert!(simple_map::contains_key(shareholders, &shareholder_address), 0);
+        assert!(simple_map::contains_key(shareholders, &shareholder_address), error::not_found(ESHAREHOLDER_NOT_EXIST));
+        let (_, shareholders_vesting) = simple_map::remove(shareholders, &shareholder_address);
+
+        // remove `shareholder_address` from `vesting_contract.beneficiaries`
+        let shareholder_beneficiaries = &mut vesting_contract.beneficiaries;
+        assert!(simple_map::contains_key(shareholder_beneficiaries, &shareholder_address), error::not_found((ESHAREHOLDER_NOT_EXIST)));
+        let (_, shareholder_baneficiary) = simple_map::remove(shareholder_beneficiaries, &shareholder_address);
```

Comment:
The second argument should be shareholder_baneficiary, not the shareholder_address.

Reply:
This change ends up getting error from prover side.

Reply:
@nizam-supraoracles , as discussed with @axiongsupra , there is a bug in the specification itself. You can ignore the comment by @axiongsupra.